### PR TITLE
neckmk3_2joints experimental setup

### PR DESCRIPTION
**Pull request by SanLordKevin**
Adding the configuration files for the neck mk3 bench setup.
There are also two files for some automatic movement using the `yarpmotogui`.

### CODE REVIEW

**Comment by SanLordKevin** [comment]
I added a short readme and one photo in the assets.

File: experimentalSetups/neckmk3_2joints/README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-### Still a WIP
\ No newline at end of file
+2 Joints Neck Setup
+===================
```
Comment (pattacini) [inline experimentalSetups/neckmk3_2joints/README.md:2]:
```suggestion
===================

```

Leave a blank line.

File: experimentalSetups/neckmk3_2joints/README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+This setup is a prototype of the neck that will be mounted on `iCub` and `ergoCub`.          
+It comprehends two motors and it can be moved using `yarprobotinterface` and `yarpmotorgui`.   
+One motor is controlled by an `amc` board and the other by an `amcbldc` board.
+All the parameters to move the setup are in the configuration files.   
+
+### Photo of the setup
```
Comment (pattacini) [inline experimentalSetups/neckmk3_2joints/README.md:8]:
```suggestion
### Picture of the setup 
```

File: experimentalSetups/neckmk3_2joints/README.md [hunk 3]
```diff
@@ -1 +1,14 @@
-### Still a WIP
\ No newline at end of file
+2 Joints Neck Setup
+===================
+This setup is a prototype of the neck that will be mounted on `iCub` and `ergoCub`.          
+It comprehends two motors and it can be moved using `yarprobotinterface` and `yarpmotorgui`.
```
Comment (pattacini) [inline experimentalSetups/neckmk3_2joints/README.md:4]:
```suggestion
It consists of two motors and can be actuated using `yarprobotinterface` and `yarpmotorgui`.   
```

**Review by pattacini — CHANGES_REQUESTED** [review]
Nice!

Just a few inline suggestions that you can commit straight away from the PR GUI (by just clicking on the button)

**Comment by SanLordKevin** [comment]
Done, thanks for the suggestions!

**Review by pattacini — APPROVED** [review]
Fine with me!
Awaiting the PR in ready for review.

**Review by valegagge — APPROVED** [review]
Hi @SanLordKevin ,
it seems fine to me.

I don't know if you already did, but to test if your configuration doesn't contain typos or some parameters are missing I suggest running `yarprobotinterface `with the parameter 
 <group name="DEBUG">
        <param name="embBoardsConnected"> 0 </param>
</group>

in the general.xml file. In this way, `yarprobotinterface ` starts and parsers all the files without checking the connected board.

Obviously, the correctness of the value of each parameter will be done using the real setup.

**Comment by pattacini** [comment]
>Obviously, the correctness of the value of each parameter will be done using the real setup.

Already tested ✅

**Comment by SanLordKevin** [comment]
> I don't know if you already did, but to test if your configuration doesn't contain typos or some parameters are missing I suggest running `yarprobotinterface `with the parameter
> 
> ```
>  <group name="DEBUG">
>         <param name="embBoardsConnected"> 0 </param>
> </group>
> ```
> 
> in the general.xml file. In this way, `yarprobotinterface ` starts and parsers all the files without checking the connected board.

Thanks! I didn't knew that!

**Pull request merged**